commands/knn.go

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package commands
+
+import (
+	"fmt"
+	"odfe-cli/client"
+	ctrl "odfe-cli/controller/knn"
+	gateway "odfe-cli/gateway/knn"
+	handler "odfe-cli/handler/knn"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	knnCommandName        = "knn"
+	knnStatsCommandName   = "stats"
+	knnStatsNodesFlagName = "nodes"
+	knnStatsNamesFlagName = "stat-names"
+)
+
+//knnCommand is base command for k-NN plugin.
+var knnCommand = &cobra.Command{
+	Use:   knnCommandName,
+	Short: "Manage the k-NN plugin short for k-nearest neighbors",
```

Comment:
I do not think "short for k-nearest neighbors" is necessary.

Reply:
Ack